numba/tests/test_builtins.py

```diff
@@ -505,6 +508,50 @@ def test_hex_npm(self):
         with self.assertTypingError():
             self.test_hex(flags=no_pyobj_flags)
 
+    def test_int_str(self, flags=nrt_no_pyobj_flags):
+        pyfunc = str_usecase
+
+        small_inputs = [
+            1234,
+            1,
+            0,
+        ]
+
+        large_inputs = [
+            123456789,
+            2222222
+        ]
+
+        args = [*small_inputs, *large_inputs]
+
+        typs = [
+            types.int8,
+            types.int16,
+            types.int32,
+            types.int64,
+            types.uint,
+            types.uint8,
+            types.uint16,
+            types.uint32,
+            types.uint64,
+            types.uintp,
+            types.ushort,
+            types.ulong,
+            types.ulonglong,
```

Comment:
These are just aliases for the unsigned ints above?

Reply:
Hum...probably.

Reply:
perhaps removing them is a good idea else it's just testing things that are already tested? `{u,}int{8,16,32,64}` would probably cover everything?

Reply:
Done!